db/db.go

```diff
@@ -80,62 +69,45 @@ func Open(dbPath string) (*DB, error) {
 	openMu.Lock()
 	defer openMu.Unlock()
 
-	dbc, err := sql.Open("sqlite3_with_hook", dbPath)
+	os.MkdirAll(filepath.Dir(dbPath), 0755)
```

Comment:
OK, I see what you're saying -- my mistake.

The directory creation should be performed up a level, which I've done here: https://github.com/otoolep/rqlite/commit/e7e212411a4831fa3b24aa3a02b102ad162ac431

That way you don't need to worry about it now.